mlflow/utils/async_logging/run_artifacts.py

```diff
@@ -0,0 +1,38 @@
+import threading
+from typing import TYPE_CHECKING, Union
+
+if TYPE_CHECKING:
+    import PIL
+
+
+class RunArtifacts:
```

Comment:
should this be singular form `RunArtifact`? in the constructor it's a single `artifact_path`